docs/category.yaml

```
@@ -104,6 +104,85 @@ paths:
           description: Unauthorized – JWT token is missing or invalid
         '403':
           description: Forbidden – User doesn't have access
+  /categories:
```

YAMLlint (1.35.1)

[error] duplication of key "/categories" in mapping

(key-duplicates)